src/main/java/com/faforever/moderatorclient/api/domain/MapService.java

```diff
@@ -115,11 +128,62 @@ public MapService(FafApiCommunicationService fafApi, MapVersionMapper mapVersion
         return result;
     }
 
+    public List<MatchmakerQueue> getAllMatchmakerQueues() {
+        log.debug("Searching for all matchmaker queues");
+        ElideNavigatorOnCollection<MatchmakerQueue> routeBuilder = ElideNavigator.of(MatchmakerQueue.class)
+                .collection();
+        List<MatchmakerQueue> queues = fafApi.getAll(MatchmakerQueue.class, routeBuilder);
+        log.debug("found {} matchmaker queues", queues.size());
+        return queues;
+    }
+
+    public List<MatchmakerQueueMapPool> getListOfBracketsInMatchmakerQueue(MatchmakerQueue queue) {
+        log.debug("Searching for all brackets in queue {}", queue.getId());
+        ElideNavigatorOnCollection<MatchmakerQueueMapPool> routeBuilder = ElideNavigator.of(MatchmakerQueueMapPool.class)
+                .collection()
+                .addFilter(ElideNavigator.qBuilder().string("matchmakerQueue.id").eq(queue.getId()))
+                .addIncludeOnCollection("mapPool")
+                .addIncludeOnCollection("mapPool.mapVersions")
+                .addIncludeOnCollection("mapPool.mapVersions.map");
+        List<MatchmakerQueueMapPool> brackets = fafApi.getAll(MatchmakerQueueMapPool.class, routeBuilder);
+        for (MatchmakerQueueMapPool bracket : brackets) {
+            System.out.println("++++++++++++++++++");
+            System.out.println(bracket.toString());
```

Comment:
log.info
If separator required put it into the same string.

Reply:
Got it